assignments/prototypes.js

```diff
@@ -37,33 +36,19 @@ function destroy () {
   * should inherit destroy() from GameObject's prototype
 */
 function CharacterStats (stats){
+  CharacterStats.prototype = Object.create(GameObject.prototype);
```

Comment:
The Object.create() will be outside of the constructor function. You're right there is one piece that goes inside (Hint: `.call`).